templates/components/multiagent/python/app/workflows/events.py

```diff
@@ -0,0 +1,27 @@
+from enum import Enum
+from typing import Optional
+
+from llama_index.core.workflow import Event
+
+
+class AgentRunEventType(Enum):
+    TEXT = "text"
+    PROGRESS = "progress"
+
+
+class AgentRunEvent(Event):
+    name: str
+    msg: str
+    event_type: AgentRunEventType = AgentRunEventType.TEXT
+    data: Optional[dict] = None
+
+    def to_response(self) -> dict:
+        return {
+            "type": "agent",
```

Comment:
if we move this to the framework, this envelope might be a blocker (as it's needed by vercel/ai) - how would the code look like if we just return:
```
{
                "agent": self.name,
                "type": self.event_type.value,
                "text": self.msg,
                "data": self.data,
            }
```

Reply:
hmm, i didn't get your idea. what is the code you want to know for that change (front-end or backend)? 
Basically if those fields is not changed to others then we can easily reformat it from the application, or we can of course can do that it front-end.